### ISSUE TITLE
Add action -> Add POI: Only predefined POI categories selectable

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/Osmand#6138]
The regular POI overlay offers a search function to select sub-categories such as "bicycle parking" or "bakery". This search function is not present in the dialog oepend via Add action -> Add POI.

**Comment by vshcherb** [comment]
If we are talking about OSM Editing:
This could be done by creating a custom filter I guess or via advanced tag/value.

**Comment by scaidermern** [comment]
No, this is (not only) about OSM editing. My description was a little bit inaccurate. I wrote "Add action -> Add POI" but actually meant "Add action -> Show/hide POI". Both actions have the same issue.

Instead of adding custom filters I would expect a search button similar to the one found at "Configure map -> POI overlay".

Configure map -> POI overlay: Starts with "Search"
![photo_2018-10-31_21-51-56](https://user-images.githubusercontent.com/1040784/47817935-3bcd3e80-dd57-11e8-95f0-a7c7bac6fe71.jpg)

"Add action -> Show/hide POI": Doesn't contain "Search"
![photo_2018-10-31_21-52-01](https://user-images.githubusercontent.com/1040784/47817934-3bcd3e80-dd57-11e8-90d4-df119c7eb3ad.jpg)
